ee/tabby-db/src/user_completions.rs

```diff
@@ -161,7 +163,7 @@ impl DbConn {
                        views,
                        IIF(language IN ({all_languages}), language, 'other') as language
                 FROM user_completions JOIN users ON user_id = users.id AND users.active
-                WHERE user_completions.created_at >= ?1 AND user_completions.created_at < ?2
+                WHERE user_completions.created_at >= ?1 AND user_completions.created_at <= ?2
```

Comment:
```suggestion
                WHERE user_completions.created_at >= ?1 AND user_completions.created_at < ?2
```

we still wanna it being `[start, end)` range - could you modify the test case to make this work?